build.sbt

```diff
@@ -64,8 +70,10 @@ lazy val root = (project in file(".")).enablePlugins(PlayScala, RiffRaffArtifact
     packageDescription := """manage tags""",
 
     Compile / doc := (target.value / "none"),
-    scalaVersion := "2.12.13",
-    ThisBuild / scalaVersion := "2.12.13",
-    libraryDependencies ++= dependencies
+    scalaVersion := scalaVer,
+    ThisBuild / scalaVersion := scalaVer,
+    libraryDependencies ++= dependencies,
+
+    gzip / includeFilter := "*.html" || "*.css" || "*.js",
```

Comment:
Just out of interest, why is this now required?

Reply:
it's not new; I moved out of `project/TagManager.scala` into the main `build.sbt` because I think having config under project is confusing & hard to find